### ISSUE TITLE
Unable to Connect via SSH

### ISSUE THREAD

**Issue by dasundev** [LuisEnMarroquin/setup-ssh-action#4]
I generated an SSH key using the following command:

```bash
ssh-keygen -t ed25519 -C "hello@dasun.dev"
```

After setting up the GitHub Action and running it, I encountered the following error:

```
[dasun]: Load key "/home/runner/.ssh/access": error in libcrypto
[dasun]: ***@***: Permission denied (publickey).
```

**Comment by LuisEnMarroquin** [comment]
Hi, it should work with any key supported, and `ed25519` should be supported, can you share your workflow file?

Else can compare with this file that I am using to test this repo on every commit: https://github.com/LuisEnMarroquin/setup-ssh-action/blob/main/.github/workflows/ubuntu.yml

**Comment by N1ebieski** [comment]
I had the same issue. In my case, the problem was with the new key format. You can recognize such a key by the fact that it has at the beginning ```-----BEGIN OPENSSH PRIVATE KEY-----```.

You need to convert this key by PuTTYgen to the old PEM format. Then it has: ```-----BEGIN RSA PRIVATE KEY-----``` and works.

**Comment by LuisEnMarroquin** [comment]
Was it on Windows? I think windows has some limitations regarding key formats

**Comment by N1ebieski** [comment]
No. Key was originally generated on Debian 10 by the command: ```ssh-keygen -t rsa -b 4096 -C "email@example.com"```.

**Comment by jetsung** [comment]
fixed:
```bash
echo "" >> ~/.ssh/access
```
